packages/tools/kda-cli/readme.md

```diff
@@ -0,0 +1,77 @@
+# kda-cli
+
+This cli tool is here to assist you with your development on the kadena blockchain.
+
+## Prerequisite
+
+In order to use this cli tool the following packages need to be installed manually.
+In a later version some might be installed via the cli tool.
+
+- [node v16+](https://nodejs.org/en)
+- [docker](https://docs.docker.com/get-docker/)
+
+Login to docker with your
+[github account](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry)
+, use the gh access token as password:
+
+```
+docker login ghcr.io -u <your gh username>
```

Comment:
I read too fast and used my password, because that's what it asked for, so it failed:

```
❯ docker login ghcr.io -u lars@webpro.nl
Password: 
Error response from daemon: Get "https://ghcr.io/v2/": denied: denied
```

Something like his might be nice to add:

```
❯ echo $GITHUB_TOKEN | docker login ghcr.io --username <your gh username> --password-stdin
Login Succeeded
```